pyiceberg/catalog/glue.py

```diff
@@ -779,3 +779,7 @@ def list_views(self, namespace: Union[str, Identifier]) -> List[Identifier]:
 
     def drop_view(self, identifier: Union[str, Identifier]) -> None:
         raise NotImplementedError
+
+    @staticmethod
+    def __is_iceberg_table(table: TableTypeDef) -> bool:
+        return table["Parameters"] is not None and table["Parameters"][TABLE_TYPE].lower() == ICEBERG.lower()
```

Comment:
nit: `pyiceberg.catalog.ICEBERG` is already lower cased

```suggestion
        return table["Parameters"] is not None and table["Parameters"][TABLE_TYPE].lower() == ICEBERG
```